assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -162,12 +169,28 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 				</p>`
 			);
 
-			if ( sourceLink && rowData[ 0 ].toLowerCase() === 'others' ) {
+			if ( sourceLink && othersSupportURL && rowData[ 0 ].toLowerCase() === 'others' ) {
```

Comment:
This is not correct. The `sourceLink` condition should not be affected, since it is for the "others" slice. This issue is about the "(other)" slice which may be present from the Analytics API response, similar to "(not set)".

So instead of doing this, you'll need to add a separate clause for "(other)", similar to the one you've added for "(not set)".

Reply:
Updated @felixarntz